### PR TITLE
Fixing documentation so manual steps lead to a working environment

### PR DESCRIPTION
**Pull request by blablacio**
Manual setup steps were quite out of date and/or lacking information.

With these changes we should be pretty much in sync with what the `test/scripts/start-services.sh` script achieves.

Not sure if the changes in `parachain/config.json` would affect staging/production builds though.

### CODE REVIEW

File: ethereum/README.md [hunk 1]
```diff
@@ -34,7 +45,7 @@ Start a ganache instance:
 
 ```bash
 ganache-cli \
-    --port=9545 \
+    --port=8545 \
```
Comment (vgeddes) [inline ethereum/README.md:48]: ah good catch :+1:

**Review by vgeddes — COMMENTED** [review]
Thanks @blablacio, this a good start, and you caught a lot of wrong/stale/outdated things.

Overall though - I think as a strategy (which hasn't been clear), I've been trying to make the E2E scripts under `test/` the preferred way to locally test and try out the system as a whole.

That allows us to simplify our individual component README's. They can just focus on documenting the component and its requirements in isolation. For example, the Ethereum README can focus on describing how to develop and test the contracts using `truffle develop`.

With that in mind, I'm not really in favour of duplicating parts of our E2E setup across our component READMEs. Some of that code/config is quite messy.

To make this strategy we work out in practice, we should also develop a separate "Deployment" doc that basically describes how to setup the entire system from scratch, similar to the E2E scripts. But we can leave that for later.

As a way forward for this PR - I also need to update the documentation after I landed #280. So I'll create a new branch and merge your improvements into them (or vice versa).

**Review by philipstanislaus — COMMENTED** [review]
LGTM, thanks for the cleanup! 🎉

File: ethereum/README.md [hunk 2]
```diff
@@ -18,6 +18,17 @@ Install Ganache
 yarn global add ganache-cli
 ```
 
+---
+**NOTE**
+
+Make sure to read [Yearn docs](https://classic.yarnpkg.com/en/docs/cli/global/) regarding global packages and path in case `Ganache` is not working.
```
[the remark this answers is not in the document]
Comment (blablacio) [reply ethereum/README.md:24]: Ooops, good catch!

File: ethereum/README.md [hunk 3]
```diff
@@ -34,7 +45,7 @@ Start a ganache instance:
 
 ```bash
 ganache-cli \
```
[the remark this answers is not in the document]
Comment (blablacio) [reply ethereum/README.md:47]: Sounds good. Go ahead or let me know if you'd want me to do the change.

[the remark this answers is not in the document]
Comment (vgeddes) [reply ethereum/README.md]: No worries I've made the necessary changes.

**Pull request merged**